### ISSUE TITLE
css.scss to .scss

### ISSUE THREAD

**Issue by crearc** [railslove/fontello_rails_converter#24]
Rails warns that css.scss is deprecated:

```
DEPRECATION WARNING: Extra .css in SCSS file is unnecessary
```

**Comment by crearc** [comment]
NVM now I see the config flag!

**Comment by crearc** [comment]
Actually, there was a problem with the options parsing. I'll submit a PR.

**Comment by jhilden** [comment]
@crearc this issue should be fixed with fc5e57751122182ee35bad058d9b633af6311800 and the release of version `0.3.3`.

**Comment by crearc** [comment]
Thank you!
